docs/sdk_example.ipynb

```diff
@@ -222,9 +225,7 @@
     "### Submitting autoRIFT jobs\n",
     "\n",
     "AutoRIFT supports processing Sentinel-1, Sentinel-2, or Landsat-8 Collection 2 pairs.\n",
-    "* Sentinel-1 jobs are submitted using [ESA granule IDs](https://sentinel.esa.int/web/sentinel/user-guides/sentinel-1-sar/naming-conventions)\n",
-    "* Sentinel-2 jobs can be submitted using either [ESA granule IDs](https://sentinel.esa.int/web/sentinel/user-guides/sentinel-2-msi/naming-convention)\n",
-    "  or [Element 84 Earth Search IDs](https://registry.opendata.aws/sentinel-2/)\n",
+    "* Sentinel-1 and Sentinel-2 jobs are submitted using [ESA granule IDs](https://sentinel.esa.int/web/sentinel/user-guides/sentinel-1-sar/naming-conventions)\n",
```

Comment:
we've lost the link to the ESA sentinel-2 naming convention here, maybe have the words `Sentinel-1` and `Sentinel-2` be links to their respective pages?